tasks/console-errors.js

```diff
@@ -0,0 +1,85 @@
+const puppeteer = require('puppeteer');
+const { getSitemapUrls } = require('./get-sitemap-links');
+
+const excludedErrors = [
+  {
+    type: 'Shortbread',
+    errorText:
+      "Shortbread failed to set user's cookie preference because the domain name that was passed"
+  }
+];
+
+const excludedScripts = [
+  'prod.assets.shortbread.aws',
+  'prod.tools.shortbread.aws',
+  'aa0.awsstatic.com'
```

Comment:
Can we be more specific in case we use some other script form aa0.awsstatic.com in the future?